specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2018-06-17-preview/workbooks_API.json

```diff
@@ -350,8 +350,7 @@
       "required": [
         "displayName",
         "category",
-        "serializedData",
-        "userId"
```

Comment:
why is this one required and readonly?

Reply:
thanks, userId is actually not required at all, I'm going to remove this